docs/package.json

```diff
@@ -6,7 +6,9 @@
     "start": "nuxt start"
   },
   "devDependencies": {
-    "@nuxt/content-theme-docs": "latest",
     "nuxt-edge": "latest"
+  },
+  "dependencies": {
```

Comment:
Shall we keep both in same section? (`dependencies` or `devDependnecies`)

Reply:
Yes of course, you are right 👍